test/integration/test_fetch_subscription_events.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from chartmogul import Config, SubscriptionEvent
+
+config = Config(api_key = '85315c0a8b2b281ccb2352790ab373fd')
+
+class FetchSubscriptionEventsTestCase(unittest.TestCase):
+    def test_subscription_events(self):
+        result = SubscriptionEvent.all(config).get()
+        print(result)
```

Comment:
We should instead use the VCR cassette approach instead of using actual requests every time. Example: 
https://github.com/chartmogul/chartmogul-python/blob/c85832c56594f610621909dbf4a4ba1edc18abf8/test/integration/test_delete_invoice.py#L69-L70

[nit] And perhaps there's a better way to assert the result instead of printing it, just to make sure that they're all valid